lib/jekyll/commands/help.rb

```diff
@@ -12,9 +12,9 @@ def init_with_program(prog)
             c.action do |args, _|
               cmd = (args.first || "").to_sym
               if args.empty?
-                puts prog
+                Jekyll.logger.info prog
```

Comment:
We might have to call `prog.to_s` here.

Reply:
done

Reply:
LogAdapter converts all parameters to strings..
https://github.com/jekyll/jekyll/blob/1691685c775f2721c1984f6632dee0fc971dfcf9/lib/jekyll/log_adapter.rb#L114-L122